vars/postReleaseToGithub.groovy

```diff
@@ -0,0 +1,12 @@
+def call(String message, String user, String repository, String credentialsToken, String branch, String tag, String artifactPath) {
```

Comment:
❔ Should we add some of the PR description as comment here? Seems like good information that we may not want to lose. Although I'm guessing this is fairly straightforward.